config.py

```
@@ -60,14 +63,101 @@ def _detect_system_memory():
 DPI_PRIMARY       = 300   # Primary DPI for PDF conversion
 DPI_ENHANCED      = 600   # Enhanced DPI for low-confidence extractions
 
+# --- Performance Optimization Settings ------------------------------------------
+# These control speed vs quality tradeoffs
+
+# Maximum image dimensions for resizing (0 to disable resizing)
+MAX_IMAGE_WIDTH   = int(os.getenv("MAX_IMAGE_WIDTH", "2000"))   # Max width in pixels
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)